toolkit/tools/scheduler/schedulerutils/printresults.go

```diff
@@ -129,89 +130,104 @@ func PrintBuildSummary(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMutex, bu
 		conflictsLogger("Number of toolchain SRPM conflicts: %d", len(srpmConflicts))
 	}
 
-	if len(builtSRPMs) != 0 {
-		logger.Log.Info("Built SRPMs:")
-		for srpm := range builtSRPMs {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
-		}
-	}
-
 	if len(testedSRPMs) != 0 {
 		logger.Log.Info("Tested SRPMs:")
-		for srpm := range testedSRPMs {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
-		}
-	}
-
-	if len(prebuiltSRPMs) != 0 {
-		logger.Log.Info("Prebuilt SRPMs:")
-		for srpm := range prebuiltSRPMs {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
+		keys := getSortedKeys(testedSRPMs)
+		for _, testedSRPM := range keys {
+			logger.Log.Infof("--> %s", filepath.Base(testedSRPM))
 		}
 	}
 
 	if len(prebuiltDeltaSRPMs) != 0 {
 		logger.Log.Info("Skipped SRPMs (i.e., delta mode is on, packages are already available in a repo):")
-		for srpm := range prebuiltDeltaSRPMs {
-			logger.Log.Infof("--> %s", filepath.Base(srpm))
+		keys := getSortedKeys(prebuiltDeltaSRPMs)
```

Comment:
For order, I'd say delta and skipped are the least important (I'm thinking least important goes at the top?), tested RPMs are probably of similar importance to built rpms...

To avoid duplication, would we want to merge built/tested? Maybe have `built only`, `tested only`, built and tested`?